cli/internal/helm/testdata/Azure/constellation-services/charts/ccm/templates/azure-daemonset.yaml

```diff
@@ -16,31 +15,32 @@ spec:
         k8s-app: cloud-controller-manager
     spec:
       containers:
-      - name: cloud-controller-manager
-        image: ccmImageForAzure
-        command:
-        - cloud-controller-manager
-        - --cloud-provider=azure
-        - --leader-elect=true
-        - -v=2
-        - --controllers=*,-cloud-node
-        - --cloud-config=/etc/azure/azure.json
-        - --allocate-node-cidrs=false
-        - --configure-cloud-routes=false
-        resources: {}
-        volumeMounts:
-        - name: etckubernetes
-          mountPath: /etc/kubernetes
-          readOnly: true
-        - name: etcssl
-          mountPath: /etc/ssl
-          readOnly: true
-        - name: etcpki
-          mountPath: /etc/pki
-          readOnly: true
-        - name: azureconfig
-          mountPath: /etc/azure
-          readOnly: true
+        - name: cloud-controller-manager
+          image: ccmImageForAzure
+          command:
+          - cloud-controller-manager
+          - --cloud-provider=azure
+          - --leader-elect=true
+          - --cluster-cidr=192.0.2.0/24
```

Comment:
This value didn't exist in the previous chart and is causing the unit tests to fail.
Should it be herer?

Reply:
Ah I didn't realize someone changed sth, thought it was just me formatting weirdly :D